.github/workflows/compiler-warnings.yml

```diff
@@ -57,7 +59,11 @@ jobs:
 
     - name: Install dependencies
       run: |
-        .github/workflows/scripts/install-build-deps.sh
+        sudo apt update && \
+        sudo apt install -y \
```

Comment:
So, you are switching to system packages instead of our own LLVM. Sound ok for checking compiler compiler warnings. But probably I'm missing the real reason for doing that.

Reply:
Not applicable anymore